drivers/dma/dma_mcux_edma.c

```diff
@@ -667,6 +696,8 @@ static int dma_mcux_edma_init(const struct device *dev)
 	IRQ_CONFIG(n, idx, dma_mcux_edma_##n##_irq_##idx)
 
 #define DMA_MCUX_EDMA_CONFIG_FUNC(n)						\
+	IF_ENABLED(DT_INST_NODE_HAS_PROP(n, channels_shared_irq_mask), \
```

Comment:
this isn't technically what IF_ENABLED is for, you should probably use cond_code_1

Reply:
Why I should cond_code_1?  I don't need _else_code on this case. 
`#define COND_CODE_1(_flag, _if_1_code, _else_code) \
	Z_COND_CODE_1(_flag, _if_1_code, _else_code)`

Reply:
okay, on second thought, you can actually use this macro here since it is checking for a 1, nevermind